<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyGenesisGUI</class>
 <widget class="QMainWindow" name="PyGenesisGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genesis Postprocessor in Python</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Datasets (RegExp)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PlotCommand"/>
      </item>
      <item>
       <widget class="QTreeWidget" name="DataBrowser">
        <column>
         <property name="text">
          <string>Files and Datasets</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="UIReplot">
          <property name="text">
           <string>Update Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Relative Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="UIPos">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UILegend">
          <property name="text">
           <string>Add Legend</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Y-Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UIYLabel"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UITitle"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Xmin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Xmax</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="UIXmin"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="UIXmax"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Ymin</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="UIYmax"/>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="UIYmin"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Ymax</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Selected Datasets</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="DatasetList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="mplwindow" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>700</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="mplvl"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionReload"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actionDuplicate"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionCor1"/>
    <addaction name="actionCor2"/>
    <addaction name="actionAutocorrelation"/>
    <addaction name="actionWigner"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <widget class="QMenu" name="menuColor_Scales">
     <property name="title">
      <string>Colormaps</string>
     </property>
     <addaction name="actionviridis"/>
     <addaction name="actionInferno"/>
     <addaction name="actionSeismic"/>
     <addaction name="actionOcean"/>
     <addaction name="actionTerrain"/>
     <addaction name="actionJet"/>
    </widget>
    <addaction name="menuColor_Scales"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuPlot"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete Dataset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate Dataset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCor1">
   <property name="text">
    <string>Temporal Coherence, 1st order</string>
   </property>
  </action>
  <action name="actionCor2">
   <property name="text">
    <string>Temporal Coherence, 2nd order</string>
   </property>
  </action>
  <action name="actionAutocorrelation">
   <property name="text">
    <string>Autocorrelation</string>
   </property>
  </action>
  <action name="actionWigner">
   <property name="text">
    <string>Wigner Distribution</string>
   </property>
  </action>
  <action name="actionviridis">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Viridis</string>
   </property>
  </action>
  <action name="actionInferno">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inferno</string>
   </property>
  </action>
  <action name="actionSeismic">
   <property name="text">
    <string>Seismic</string>
   </property>
  </action>
  <action name="actionOcean">
   <property name="text">
    <string>Ocean</string>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="text">
    <string>Terrain</string>
   </property>
  </action>
  <action name="actionJet">
   <property name="text">
    <string>Jet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>PyGenesisGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
